diff --git a/lib/modules/__init__.py b/lib/modules/__init__.py
index be912e3..ea0d799 100644
--- a/lib/modules/__init__.py
+++ b/lib/modules/__init__.py
@@ -129,6 +129,7 @@ def all_modules() -> list[type[Module]]:
     from lib.modules.bcr import BCRModule
     from lib.modules.custota import CustotaModule
     from lib.modules.msd import MSDModule
+    from lib.modules.norththermrewire import NorthThermRewireModule
     from lib.modules.oemunlockonboot import OEMUnlockOnBootModule
 
     return [
@@ -136,5 +137,6 @@ def all_modules() -> list[type[Module]]:
         BCRModule,
         CustotaModule,
         MSDModule,
+        NorthThermRewireModule,
         OEMUnlockOnBootModule,
     ]
